source/common/config/delta_subscription_impl.h

```diff
@@ -210,10 +214,21 @@ class DeltaSubscriptionImpl
       init_fetch_timeout_timer_.reset();
     }
   }
+
+  // Use these helpers to avoid forgetting to update both at once.
+  void insertOrUpdateResourceVersion(const std::string& key, const std::string& val) {
```

Comment:
Nit: prefer `absl::string_view` in all new code over `const std::string&`, it's strictly more powerful.

Reply:
It interferes with unordered_map. It sounds like it would need a hash explicitly defined. Since this isn't particularly performance sensitive code, I think it's not worth it.